app/models/shelter.rb

```diff
@@ -31,4 +37,8 @@ def alphabetical_pets
   def shelter_pets_filtered_by_age(age_filter)
     adoptable_pets.where("age >= ?", age_filter)
   end
+
+  def self.order_by_name_reverse
```

Comment:
Nice! This is where this shelter query should live (you can probably delete the admin model you have)